README.md

```diff
@@ -134,19 +148,21 @@ Setting to 'absent' deletes the device from `device.conf` and the `devices` fact
 
 Data type: String
 
-Specifies the type of the device in `device.conf` on the proxy Puppet agent.
+Specifies the type of the device in `device.conf` on the proxy Puppet agent. This is the module type to be used to access the device.
 
 ### url
 
 Data type: String
 
+This parameter is optional - either credentials or url must be used, however these are mutually exclusive (only one must be used).
+
```

Comment:
This is a bit confusing, isn't the second part of the sentence saying the same thing as the first half?

Switch to active voice "use either the credentials or url".

Also should it be url or URL?

Reply:
Changed. I've kept the parameter title in lowercase, but reference to it as uppercase?

"This parameter is optional - use either the credentials or URL." - for url

"This parameter is optional and specific to devices that use the Puppet Resource API. Use either the credentials or URL." - for credentials. I kept in the existing note about Resource API.

Thanks!